ArgoProject/src/main.cpp

```diff
@@ -13,6 +13,11 @@ int main(int argc, char** argv)
 		app::Console::writeLine({ "ERROR: SDL_ttf Failed to initialize [", TTF_GetError(), "]" });
 		return EXIT_FAILURE;
 	}
+	if (Mix_OpenAudio(44100, MIX_DEFAULT_FORMAT, 2, 2048) < 0)
```

Comment:
you are missing the function call `Mix_Init()` that should be run before this call.
```suggestion
	if (Mix_OpenAudio(MIX_DEFAULT_FREQUENCY, MIX_DEFAULT_FORMAT, MIX_DEFAULT_CHANNELS, 2048) != NULL)
```